.pre-commit-config.yaml

```diff
@@ -71,7 +72,12 @@ repos:
     rev: "v2.5.1"
     hooks:
       - id: prettier
-        types: ["markdown"]
+        types_or: ["markdown", "json"]
```

Comment:
It's nicer to have this diff-friendly:
```suggestion
        types_or:
        - markdown
        - json
```

Reply:
ditto, if formatter didn't do it, i don't care about it.

Reply:
Well, I do — lists are naturally multi-line entries.